docs/apps/components/cascader.md

```diff
@@ -0,0 +1,195 @@
+|`this.hide(fields)`|Hides the field|`(fields: String(componentID)`|
+|`this.show(fields)`|Displays the field|`(fields: String(componentID)`|
+| `this.disable(fields)`| Disable input field from user interaction|`(fields: String(componentID), String[])`|
+
+|`this.setData(Value)`|Set the data in the field|
+|**this.refreshFieldDataSource**|Refresh the datasource data bound to the form field|
+|**this.setOptions(fields, options)**|Set Form Field Configuration Item|
+
+!!! Info
+    1. The show() and hide() methods can also be used to control the visibility of an alert in response to user input.
+    2. **fields** refer to the component **ID**. You can fetch the ID from the **Component Attribute** panel in the Page Builder.
+    3. For using **this.show(fields)** enable the **Hidden** check-box in the **Component Attribute** panel.
+
+#### Steps to place/use the methods for components of the Page Builder"
+
+1. Go to Form Attribute :material-menu-right: Action Panel :material-menu-right: Setting :material-menu-right: (Mounted | refresh | click 'Add action')
+2. Write the method/code as shown in the **Example** below
+3. Click on `Save`.
+4. On the main screen click on `Save` again.
+5. Click on `Preview` to see the code in action.
+<img src= "/apps/components/img/alert1.png">
+
+!!! Example
+    1. `this.addClassName(fields, className)`
+       * Go to Form Attribute :material-menu-right: Style Sheets :material-menu-right: add the class
+        ```js
+        .abc{ // abc is the class name
+        background-color: red;
+        }
+        ```
+       * Follow the steps in the above **Note**
+       * ```js
+         this.addClassName('button_815t5dfs', 'abc')
+         ```
+
+    2. `this.removeClassName(fields, className)`
+    ```js
+    this.removeClassName('button_815t5dfs', 'abc')
+    ```
+    
+    3. `this.getValue(fieldName)`
+    ```js
+    var buttonname = this.getValue('button_815t5dfs');
+    console.log('getValue', buttonname);
+    ```
+    4. `this.getValues(fieldName)`
+    ```js
+    var data = this.getValues();
+    console.log(data);
+    ```
+    5. `this.hide(fields)`
+        ```js
+        var fields= ['button_815t5dfs']
+        this.hide(fields)
+        ```
+    6. `this.show(fields)`
+        ```js
+        var fields= ['button_815t5dfs']
+        this.show(fields)
+        ```
+    7. `this.disable(['fields'])`
+        ```js
+        this.disable(['button_6ytj0ne9'])
+        ```
+
+## Config
+
+| **Name**|**Description**|**Image**|
+|---------------|----------------------------------------------------------------------------------------------------------------------------------------|---|
+|**ID**| Cascader's unique identifier|<img src= "/apps/components/img/input_id.png">|
+|**Name**| [**Optional**] The visual identifier label of the field|<img src= "/apps/components/img/checkbox_name.png">|
+|**Width**| [**Optional**] To set the width of the field|<img src= "/apps/components/img/input_width.png">|
```

Comment:
Improve wording here too please. To set sounds like an action and not a description.

Reply:
done
The width of the field